llvm/test/CodeGen/X86/call-graph-section.ll

```diff
@@ -0,0 +1,50 @@
+;; Tests that we store the type identifiers in .callgraph section of the binary.
+
+; RUN: llc --call-graph-section -filetype=obj -o - < %s | \
+; RUN: llvm-readelf -x .callgraph - | FileCheck %s
```

Comment:
Should there be dedicated assembler and disassembler tests in test/MC?

Reply:
Added a separate test to test the .S output. PTAL.

Reply:
I don't see any tests under `tests/MC` though. I agree w/ @arsenm that we need to have dedicated tests for the assembler/disassembler, and not just for codegen. Those commonly find bugs in the implementation that are otherwise easy to miss.

Reply:
Added a new llvm-mc test. Please take a look.

Reply:
Marking this as done.